src/App.js

```diff
@@ -1,25 +1,13 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import InputColumn from "./Component/InputColumn";
+import "./style.css";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+export default class App extends Component {
+  render() {
+    return (
+      <div>
+        <InputColumn/>
```

Comment:
If this will be your only component as the main component for form, you can name it better